### ISSUE TITLE
Jagex bug - Some items cannot be withdrawn for 1 tick after withdrawing certain items.

### ISSUE THREAD

**Issue by geheur** [geheur/bank-tag-custom-layouts#71]
When an item that does not leave a placeholder (such as a clue scroll) is the last item in a bank tab and is withdrawn from the bank, all items in later tabs (tab order being 1 2 3 4 5 6 7 8 9 infinity), cannot be withdrawn for 1 tick.

Make sure items such as this are stored in the infinity tab to entirely avoid this issue.
